packages/@react-spectrum/s2/intl/nb-NO.json

```diff
@@ -12,16 +13,16 @@
   "label.(optional)": "(valgfritt)",
   "label.(required)": "(obligatorisk)",
   "menu.moreActions": "Flere handlinger",
-  "picker.placeholder": "Velg et alternativ...",
+  "picker.placeholder": "Velg …",
```

Comment:
Space before ellipses was removed below but added here?

Reply:
This is consistent with other "Velg …" so used here. For this PR will keep but do overall review with vendor to fix these for future update. For example, there are cases of:
- Laster inn ...
- Laster inn...

I need to check with translators as almost 50% of each type